### ISSUE TITLE
Loading time observations on the map

### ISSUE THREAD

**Issue by KarelSchoonvaere** [inbo/vespa-watch#292]
Either it takes forever to open an observation on the homepage map or it doesn't work. You can try to reproduce this by zooming in on the map and clicking a random observation (orange circle). You'll see the message "loading" but actually the pop-up never comes. 

This is a crucial part of the interactivity with citizens, people have to be able to scroll down to their region and click on the observation to get more information.

**Comment by peterdesmet** [comment]
I had this on my phone yesterday. Where did you encounter the problem? We’ll try to replicate.

@niconoe

**Comment by KarelSchoonvaere** [comment]
On my desktop at home, via Chrome browser

**Comment by niconoe** [comment]
Yep, I have difficulty replicating this which is indeed a problem. Next time someone encounters it, would it be possible to:

- Note the exact date and time so we can have a look at the server logs
- Open your browser's javascript console and copy paste any error there?

thx!

**Comment by KarelSchoonvaere** [comment]
I encounter it all the time. 
27-05-2019 12:01

Javascript console ( output of CTRL+SHIFT+J):
Access to resource at 'https://lw-vespawatch-prd.s3.amazonaws.com/static/vespawatch/icons/site.webmanifest' from origin 'https://vespawatch.be' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

![image](https://user-images.githubusercontent.com/26611593/58412818-8dcd0d00-8077-11e9-8604-a9a5efe45751.png)

**Comment by stijnvanhoey** [comment]
Is it on a Mac product?

![image](https://user-images.githubusercontent.com/754862/58416024-97a73e00-8080-11e9-9119-453fc69d48e4.png)

**Comment by bartaelterman** [comment]
I'm on a mac. I don't have an issue with the CORS policy

**Comment by KarelSchoonvaere** [comment]
Windows, rolled out by Ghent Uni

**Comment by bartaelterman** [comment]
@stijnvanhoey do we have some kind of whitelist for other domains? The images are on lw-vespawatch-prd.s3.amazonaws.com so maybe some browsers don't allow loading these images by default because they don't come from the domain vespawatch.be.
Otherwise, maybe we can add that with [this package](https://pypi.org/project/django-cors-headers/)? @niconoe ?

**Comment by bartaelterman** [comment]
I would've expected django to fix the CORS setting of static files by default. But I can't find a `Access-Control-Allow-Origin` header in the response from django. So I guess we need to make sure that one is returned.

**Comment by niconoe** [comment]
@bartaelterman:

- I think I've worked once with the package you mentioned for CORS, I don't remember any issue with it.
- Are we sure this CORS error is the source of this specific problem? I got the error all the time in my browser, but the map interactions work fine.
- Also, since this error occur on a static files, is it really served by Django in production (might directly comes from Apache/Nginx => check how deployment works).

**Comment by stijnvanhoey** [comment]
@bartaelterman we do have the concept of allowed hosts: https://github.com/inbo/vespa-watch/blob/master/djangoproject/settings/settings.py#L14 o the django-side if that is what you mean with whitelisting?

**Comment by bartaelterman** [comment]
@stijnvanhoey good suggestion, we could try that before installing the CORS package.
@niconoe indeed, we don't know whether that's the issue.

**Comment by niconoe** [comment]
Issue still happening?

**Comment by peterdesmet** [comment]
Popups load fine on my phone now. CORS issue does not seem related. @niconoe do you want to create a separate issue for CORS or repurpose this one?
